src/qt/clientmodel.h

```diff
@@ -127,7 +125,6 @@ public Q_SLOTS:
     void updateTimer();
     void updateMnTimer();
     void updateNumConnections(int numConnections);
-    void updateAlert();
```

Comment:
Shouldn't we connect `AlertNotify/NotifyAlertChanged` with the GUI, rather than removing these?

Reply:
yeah, we should. I removed it because we were broadcasting the signal several times in row but can add it now, the problem is fixed in e074feb.
Will connect it.